src/components/blocks/DlLabelPicker/DlLabelPicker.vue

```diff
@@ -188,6 +197,16 @@ export default defineComponent({
             }
             return false
         }
+
+        onMounted(() => {
+            nextTick(() => {
+                const target = table.value.$el.querySelector(
+                    `[data-label-picker-identifier="${items.value?.[0]?.identifier}"]`
```

Comment:
Check that `items.value?.[0]?.identifier` exists first, so we don't update the `selected` class when the identifier is not defined